test/stepDefinitions/caseDetailsSteps.js

```diff
@@ -97,9 +97,12 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
   });
 
   When(/^I click on its first accordion on the '(.*)' tab$/, async function (tabName) {
-    await caseDetailsPage.clickTab(tabName);
-    await caseDetailsPage.clickFirstAccordian();
+      await caseDetailsPage.clickTab(tabName);
+      await caseDetailsPage.clickFirstAccordian();
   });
 
+  When(/^I progress to the next event '(.*)'$/, async function(event) {
```

Comment:
looking at the code, should't this be 'I start the event' or 'I trigger the event'?

Reply:
this is just being picky but I will give it to you